go.mod

```diff
@@ -5,11 +5,14 @@ go 1.18
 require (
```

Comment:
Some of the other additions of dependency logic have a separate `go.mod` under the `dependency` directory. This is good for a couple of reasons:
1. The buildpack itself doesn't pull in libraries that are only used by the dependency code
2. It is more obvious to humans/tooling which libraries are used by the buildpack vs the dependency code.

Do you have any thoughts on this? I'd like to split them if possible.

Reply:
This is a good idea. I am happy to split it up.